### DISCUSSION TITLE
Welcome to znote-app Discussions!

### DISCUSSION THREAD

**Discussion by alagrede** [alagrede/znote-app#9] [General]
## 🧪 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dcerniglia** [comment]
Hi, @alagrede ! I was super excited to find Znote yesterday. This is exactly what I've been looking for, and I would love to get involved. I'm a junior developer who has never contributed to a project like this, but I would like to contribute in any way I can.

**Reply by alagrede** [reply]
Thank you so much for your interest 🙏. Currently I'm looking for feedback more than code contribution.
If you have some libraries or plugins ideas to integrate,I will be very interested to discuss with you.
Again I am open to all proposals. So don't hesitate to discuss with me.

**Reply by dcerniglia** [reply]
I would like to see VIM keybindings at some point. The ability to customize the theme or use VS-Code-style themes, would be cool as well.
